### ISSUE TITLE
Render new note while view the graph

### ISSUE THREAD

**Issue by LintaoAmons** [agerardin/joplin-plugin-knowledge-graph#3]
While viewing the graph(filter by tag, like `java`), I create a note which has the `java` tag.

It won't appeal directly to the graph and is linked to the `java` node.

**Comment by LintaoAmons** [comment]
Same, When I add new tag to a note, it won't link to the tag node instantly

**Comment by agerardin** [comment]
Thanks for your feedback!
Indeed this is how the UI reacts for now and this is not a bug per say but related to Joplin's implementation.

Tags and notes and handled separately in Joplin,
so adding a tag to a note won't trigger a notification of change for the note
and the plugin is not aware of it until a full refresh is requested (this will happen when another note is selected).

I have not yet figure out a way to receive notifications for tag updates, without polling the data API constantly.
However it may well be the correct way to overcome various limitations
of the notification API, so I may end up doing that.

If you know of another plugin that solves this issue properly, please let me know so I can have a look!

**Comment by imthevs** [comment]
Here's a wild idea - how about creating notes and tags while you are in the graph view?  I can imagine using this as a mindmap!

**Comment by agerardin** [comment]
Indeed! I am currently focusing on improving performance for the main use case, but that is the kind of things I have in mind moving forward. Stay tuned!

**Comment by agerardin** [comment]
@LintaoAmons 
The new release v1.1.0 has addressed this issue.
The minimum refresh time is now set to 1s. This can be modified in the settings as well using  UPDATE_INTERVAL
